lerna.json

```diff
@@ -3,8 +3,8 @@
   "npmClient": "npm",
   "packages": [
     "src/components/*",
-    "src/interfaces",
-    "src/enums"
+    "src/enums",
+    "src/interfaces"
   ],
```

Comment:
I did not add the new mixins folder to this (and it seems to work fine). As explained in another comment, my new component uses `@mixins` which grabs the local mixins folder (either here in bcrs-shared-components or when imported in our main web apps).

Reply:
I might be a good idea to publish/version the mixins folder, so as to keep changes managed. However, I don't expect UIs to import these mixins (although they could).